src/convertSchema.test.ts

```diff
@@ -332,4 +392,186 @@ describe('convertSchema', () => {
       })
     ).toThrow('Duplicate paths found: name')
   })
+  describe('the `mapSchema` option', () => {
+    it('can be a no-op', () => {
+      expect(
+        convertSchema(schema, '"doc"."foobar"', { mapSchema: ({ val }) => val })
+      ).toEqual(
+        `CREATE TABLE IF NOT EXISTS "doc"."foobar" (
+  "id" TEXT PRIMARY KEY,
+  "name" TEXT,
+  "description" TEXT,
+  "numberOfEmployees" TEXT,
+  "notificationPreferences" ARRAY (
+    TEXT
+  ),
+  "addresses" ARRAY (
+    OBJECT(DYNAMIC) AS (
+      "address" OBJECT(DYNAMIC) AS (
+        "street" TEXT,
+        "city" TEXT,
+        "county" TEXT,
+        "state" TEXT,
+        "zip" TEXT,
+        "country" TEXT,
+        "latitude" BIGINT,
+        "longitude" BIGINT
+      ),
+      "name" TEXT,
+      "isPrimary" BOOLEAN
+    )
+  ),
+  "integrations" OBJECT(DYNAMIC) AS (
+    "stripe" OBJECT(DYNAMIC) AS (
+      "priceId" BIGINT,
+      "subscriptionStatus" TEXT
+    )
+  ),
+  "metadata" OBJECT(IGNORED)
+) WITH (column_policy = 'dynamic')`
+      )
+    })
+    it('can replace a leaf node', () => {
```

Comment:
Let's remove all other tests related to the `mapSchema` option other than this one. We're just testing that `map` from `obj-walker` works with the tests, which isn't necessary.